gridscale/datasource_gridscale_server_test.go

```diff
@@ -0,0 +1,45 @@
+package gridscale
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccdataSourceGridscaleServer_basic(t *testing.T) {
+	name := fmt.Sprintf("object-%s", acctest.RandString(10))
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckGridscaleIpv4DestroyCheck,
```

Comment:
`testAccCheckGridscaleIpv4DestroyCheck`  should be `testAccCheckResourceGridscaleServerDestroyCheck`. Plz, would you fix this for `TestAccdataSourceGridscaleFirewall_basic` https://github.com/terraform-providers/terraform-provider-gridscale/blob/fa59c99170c7a502363eb74884edfe04eb447606/gridscale/datasource_gridscale_firewall_test.go#L17

Reply:
Thanks for splitting https://github.com/terraform-providers/terraform-provider-gridscale/pull/48

Reply:
Plz, would you fix this for `TestAccdataSourceGridscaleFirewall_basic` https://github.com/terraform-providers/terraform-provider-gridscale/blob/fa59c99170c7a502363eb74884edfe04eb447606/gridscale/datasource_gridscale_firewall_test.go#L17

Reply:
@itakouna Done!